bliss/models/prior.py

```diff
@@ -142,23 +153,38 @@ def sample_prior(
         is_on_array = get_is_on_from_n_sources(n_sources, self.max_sources)
         locs = self._sample_locs(is_on_array)
 
-        galaxy_bools, star_bools = self._sample_n_galaxies_and_stars(is_on_array)
-        galaxy_params = self._sample_galaxy_params(galaxy_bools)
+        galaxy_bools, star_bools, lensed_galaxy_bools = self._sample_n_galaxies_and_stars(
+            is_on_array
+        )
```

Comment:
I would make a new function `_sample_n_lens` or something like that which takes `is_on_array` and `galaxy_bool` and returns `lensed_galaxy_bools`